.evergreen/config.in.yml

```diff
@@ -1168,6 +1169,50 @@ tasks:
         params:
           file: src/results.json
 
+  - name: run-spec-benchmark-tests-node-server-timeoutMS-120000
+    tags:
+      - run-spec-benchmark-tests
+      - performance
+    exec_timeout_secs: 3600
+    commands:
+      - command: expansions.update
+        type: setup
+        params:
+          updates:
+            - { key: NODE_LTS_VERSION, value: v22.11.0 }
```

Comment:
Any reason we're changing the NODE version here?

Reply:
https://github.com/mongodb/node-mongodb-native/pull/4304/files#diff-c8f20cda7687f898b104a8f147d112896ad5e15bbc2de2338bcebe2ab5b65696R1161

We updated the Node version, 18.16.0 had a bug in it that prevented us from running the benchmarks